.jshintrc

```diff
@@ -1,32 +1,31 @@
 {
-    "predef": [
-        "document"
-    ],
-    "node" : true,
-    "esnext": true,
-    "boss" : false,
-    "curly": false,
-    "debug": false,
-    "devel": false,
-    "eqeqeq": true,
-    "evil": true,
-    "forin": false,
-    "immed": true,
-    "laxbreak": false,
-    "newcap": true,
-    "noarg": true,
-    "noempty": false,
-    "nonew": true,
-    "nomen": false,
-    "onevar": false,
-    "plusplus": false,
-    "regexp": false,
-    "undef": true,
-    "sub": false,
-    "white": false,
-    "eqeqeq": false,
```

Comment:
this refrence also have in line https://github.com/mlrawlings/minprops/pull/2/commits/17ddc089869d1319a62a14598674f93243e95689#diff-4d5aa81bf4f18104bb6ea53a8b5d1f43L11